src/pages/view-test-result/cat-a-mod2/view-test-result.cat-a-mod2.page.ts

```diff
@@ -132,7 +129,7 @@ export class ViewTestResultCatAMod2Page extends BasePageComponent implements OnI
       date: startDate.format('dddd Do MMMM YYYY'),
       time: startDate.format('HH:mm'),
       applicationReference: formatApplicationReference(this.testResult.journalData.applicationReference),
-      category: TestCategory.BE,
+      category: TestCategory.EUAM2,
```

Comment:
No matter which Mod 2 test you look-up it will always display this category on the page